DigitalExcellence/dex-frontend
# Clean up layout for the embed project page.

## Description

Fixes broken layout for the project embed page. Currently the navigation and footer are hidden by listening to router events, this should be done otherwise, see #223.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Check out this branch
2. Make a project embeddable. You must own or create a project for this
3. Go to the URL of that project's embed page and set your viewport width to a small amount, such as 600px or 400px.

## Link to issue

Closes: #223
